src/main/src/main/java/org/geoserver/catalog/event/CatalogListener.java

```diff
@@ -22,6 +22,9 @@
  */
 public interface CatalogListener {
 
+    /** Handles the event of an addition to the catalog. */
```

Comment:
This comment hould not be something along the lines: ``Handles the event of an object that is about to be added tot he catalog, giving a chance to listeners to change it``.

Reply:
agreed, will do